From ffcafea3a18b34d203f3262150fb0fd48ae67c0c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20K=C3=B6gl?= <stefan@skoegl.net>
Date: Mon, 30 Jun 2014 22:37:26 +0200
Subject: [PATCH] [Setup] use utf-8 explicitly in setup.py, fixes #8

---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 59124fd..36dc457 100644
--- a/setup.py
+++ b/setup.py
@@ -2,11 +2,12 @@
 
 from distutils.core import setup
 import re
+import io
 import os.path
 
 dirname = os.path.dirname(os.path.abspath(__file__))
 filename = os.path.join(dirname, 'jsonpointer.py')
-src = open(filename).read()
+src = io.open(filename, encoding='utf-8').read()
 metadata = dict(re.findall("__([a-z]+)__ = '([^']+)'", src))
 docstrings = re.findall('"""(.*)"""', src)
 
-- 
1.9.3

